src/api/orcidApi.ts

```diff
@@ -0,0 +1,10 @@
+import { authenticatedApiRequest2 } from './apiRequest';
+import { OrcidApiPath } from './apiPaths';
+import { OrcidCredentials } from '../types/orcid.types';
+
+export const postOrcidCredentials = async (orcidCredentials: OrcidCredentials) =>
+  await authenticatedApiRequest2({
```

Comment:
Du har litt uflaks med å komme inn rett etter vi har introdusert `authenticatedApiRequest2` 😬 
Den gamle versjonen glatter over alle feil slik at man selv må håndtere statuskoder. Det var er dårlig designvalg, og passer også dårlig når vi bruker `react-query`, derfor kom denne 2. versjonen som gjør det enklere å håndtere respnser som man vil. Siden dette du legger til er midt oppe i mye annet som fins og ikke nødvendivis er verdt å bruke tid på å skrive om akkurat no tror jeg det er bedre å bruke den gamle versjonen her slik at alle responser kan håndters likt der. Messy, I know, but that's life </3 

```suggestion
  await authenticatedApiRequest({
```

Min påstand er i hvert fall at denne 2-eren ikke vil takle `isErrorStatus` etterpå når kallet feiler